### DISCUSSION TITLE
Click the back button from the tab it will return JSON data, but I want to load the page.

### DISCUSSION THREAD

**Discussion by Vishwaas-S** [adonisjs/core#3278] [Help]
In Adonis js, If I click the back button from the tab it will return JSON data. it has to load the page but instead, it returns JSON data.

How can I solve the issue?

**Comment by RomainLanz** [comment]
Hey @Vishwaas-S! 👋🏻 

It will be difficult to help you without looking at any code.

**Reply by Vishwaas-S** [reply]
if I am click  the back button from the tab

![Screenshot (66)](https://user-images.githubusercontent.com/85667787/139001390-ce553c76-c6c1-43fa-9b77-f73d7210fe2d.png)

and return json data , but I want to load the page

{
    data : { }
    index : { }
}

**Reply by feniljariwala82** [reply]
Can you please share your workflow?
